liferay-echo/liferay-portal
# LPS-121917 Migrate v2 usages in site-navigation/site-navigation-item-selector-web

The <clay:management-toolbar-v2 /> tag has been replaced with the
<clay:management-toolbar /> tag (which uses clay v3).

Selecting Navigation Menus should work as expected.

![](https://user-images.githubusercontent.com/5572/109821118-f32f3400-7c35-11eb-9a2f-fa5425c0a5a6.gif)

**Test plan**:
- Navigate to **Site Builder** > **Navigation Menus**
- Create a new Navigation Menu
- Navigate to the **Home Page**
- Right click on the Home Page's Navigation Menu
- Select **Configuration**
- In the modal window, select the **Choose Menu** radio button
- Click on the **Select** button
- In the nested modal window, make sure you can filter and select a Navigation Menu
